pvactools/tools/compare.py

```diff
@@ -0,0 +1,231 @@
+from .pvaccompare.compare_tools import *
+from .pvaccompare.server import main as startserver
+import argparse
+import logging
+from datetime import datetime
+import sys
+import os
+
+logging.basicConfig(level=logging.DEBUG, format="%(message)s")
+
+
+def define_parser():
+    """
+    Purpose:    Define arguments for the parser that the user can use
+    Modifies:   Nothing
+    Returns:    The parser
+    """
+    valid_aggregated_columns = [
+        "Gene",
+        "AA Change",
+        "Num Passing Transcripts",
+        "Best Peptide",
+        "Best Transcript",
+        "Num Passing Peptides",
+        "IC50 MT",
+        "IC50 WT",
+        "%%ile MT",
+        "%%ile WT",
+        "RNA Expr",
+        "RNA VAF",
+        "DNA VAF",
+        "Tier",
+    ]
+    valid_unaggregated_columns = [
+        "Biotype",
+        "Median MT IC50 Score",
+        "Median WT IC50 Score",
+        "Median MT Percentile",
+        "Median WT Percentile",
+        "WT Epitope Seq",
+        "Tumor DNA VAF",
+        "Tumor RNA Depth",
+        "Tumor RNA VAF",
+        "Gene Expression",
+        "BigMHC_EL WT Score",
+        "BigMHC_EL MT Score",
+        "BigMHC_IM WT Score",
+        "BigMHC_IM MT Score",
+        "MHCflurryEL Processing WT Score",
+        "MHCflurryEL Processing MT Score",
+        "MHCflurryEL Presentation WT Score",
+        "MHCflurryEL Presentation MT Score",
+        "MHCflurryEL Presentation WT Percentile",
+        "MHCflurryEL Presentation MT Percentile",
+        "MHCflurry WT IC50 Score",
+        "MHCflurry MT IC50 Score",
+        "MHCflurry WT Percentile",
+        "MHCflurry MT Percentile",
+        "MHCnuggetsI WT IC50 Score",
+        "MHCnuggetsI MT IC50 Score",
+        "MHCnuggetsI WT Percentile",
+        "MHCnuggetsI MT Percentile",
+        "NetMHC WT IC50 Score",
+        "NetMHC MT IC50 Score",
+        "NetMHC WT Percentile",
+        "NetMHC MT Percentile",
+        "NetMHCcons WT IC50 Score",
+        "NetMHCcons MT IC50 Score",
+        "NetMHCcons WT Percentile",
+        "NetMHCcons MT Percentile",
+        "NetMHCpan WT IC50 Score",
+        "NetMHCpan MT IC50 Score",
+        "NetMHCpan WT Percentile",
+        "NetMHCpan MT Percentile",
+        "NetMHCpanEL WT Score",
+        "NetMHCpanEL MT Score",
+        "NetMHCpanEL WT Percentile",
+        "NetMHCpanEL MT Percentile",
+        "PickPocket WT IC50 Score",
+        "PickPocket MT IC50 Score",
+        "PickPocket WT Percentile",
+        "PickPocket MT Percentile",
+        "SMM WT IC50 Score",
+        "SMM MT IC50 Score",
+        "SMM WT Percentile",
+        "SMM MT Percentile",
+        "SMMPMBEC WT IC50 Score",
+        "SMMPMBEC MT IC50 Score",
+        "SMMPMBEC WT Percentile",
+        "SMMPMBEC MT Percentile",
+        "DeepImmuno WT Score",
+        "DeepImmuno MT Score",
+        "Problematic Positions",
```

Comment:
Version 5.4 introduces a new features that would add a `Gene of Interest` column here that might be interesting to compare. The possible values are `True`/`False`.